scripts/genereer_opzet_woningwaarderinggroep.py

```diff
@@ -51,9 +45,15 @@ def check_write(file_path: Path) -> bool:
 questions = [
     inquirer.List(
         "stelsel",
-        message="Welk stelsel wil je implementeren?",
+        message="Voor welk stelsel wil je een stelselgroep implementeren?",
         choices=[(stelsel.naam, stelsel.name) for stelsel in Woningwaarderingstelsel],
     ),
+    inquirer.Text(
+        "begindatum",
+        message=lambda answers: f"Vanaf welke datum is het {Woningwaarderingstelsel[answers.get('stelsel')].naam} stelsel geldig?",
+        default=date(date.today().year, 7, 1),
```

Comment:
```suggestion
        default=date(date.today().year, 1, 1),
```
eigenlijk moet defaultdate voor deze package nog op januari toch?